injector/src/main/java/com/sygic/tools/processor/Processor.java

```diff
@@ -0,0 +1,283 @@
+package com.sygic.tools.processor;
```

Comment:
Tento file jsi psal celý ty nebo jsi se odněkud inspiroval? :)

Reply:
pisal som ho ja, lebo je to nas speci use case, ale jasne inspiroval som sa inymi anotacnymi procesormi ;)